CHANGELOG.md

```diff
@@ -2,6 +2,55 @@
 
 ## Unreleased
 
+## 1.5.1
```

Comment:
Should this be 1.6.0 if there's a new feature?

Reply:
Are we following https://semver.org/ ? If so then I think it's a minor change since everything should be backwards compatible.

```
Given a version number MAJOR.MINOR.PATCH, increment the:

MAJOR version when you make incompatible API changes,
MINOR version when you add functionality in a backwards-compatible manner, and
PATCH version when you make backwards-compatible bug fixes.
Additional labels for pre-release and build metadata are available as extensions to the MAJOR.MINOR.PATCH format.
```

Reply:
Oh sorry, just read the actual number and you're right. This should be 1.6.0.